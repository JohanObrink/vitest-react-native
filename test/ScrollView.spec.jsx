import { test, expect } from 'vitest'
import { render } from '@testing-library/react-native'
import { ScrollView, Text } from 'react-native'

const View = () => {
  return (
    <ScrollView>
      <Text>Hello, world!</Text>
    </ScrollView>
  )
}

test('scroll view components render correctly', () => {
  const container = render(<View />)
  expect(container).toMatchSnapshot()
})