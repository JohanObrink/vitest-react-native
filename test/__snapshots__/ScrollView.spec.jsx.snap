// Vitest Snapshot v1

exports[`scroll view components render correctly 1`] = `
<RCTScrollView>
  <View>
    <Text>
      Hello, world!
    </Text>
  </View>
</RCTScrollView>
`;
